app/views/admin/splits/_details.html.erb

```diff
@@ -19,7 +19,13 @@
       </tr>
       <tr>
         <td>Status</td>
-        <td class="u-status--<%= @split.finished? ? 'finished' : 'active' %>"><%= @split.finished? ? 'Finished' : 'Active' %><span class="sc-p--s">🏁</span></td>
+        <% if @split.finished? %>
+               <td class="u-status--finished">Finished (retired)<span class="sc-p--s">🏁</span></td>
```

Comment:
I'll be the one to nitpick on using two-space tabs in ERB templates.

Reply:
fine :)